Wrappers/Python/cil/optimisation/algorithms/PDHG.py

```diff
@@ -209,22 +224,15 @@ def set_up(self, f, g, operator, tau=None, sigma=1., initial=None, **kwargs):
         except AttributeError:
             pass        
 
-        if self.gamma_g is not None:
-            warnings.warn("Primal Acceleration of PDHG: The function g is assumed to be strongly convex with parameter `gamma_g`. Need to be sure that gamma_g = {} is the correct strongly convex constant for g. ".format(self.gamma_g))
+        if self.gamma_g is not None:            
+            warnings.warn("Primal Acceleration of PDHG: The function g is assumed to be strongly convex with positive parameter `gamma_g`. Need to be sure that gamma_g = {} is the correct strongly convex constant for g. ".format(self.gamma_g))
         
         if self.gamma_fconj is not None:
-            warnings.warn("Dual Acceleration of PDHG: The convex conjugate of function f is assumed to be strongly convex with parameter `gamma_fconj`. Need to be sure that gamma_fconj = {} is the correct strongly convex constant".format(self.gamma_fconj))
+            warnings.warn("Dual Acceleration of PDHG: The convex conjugate of function f is assumed to be strongly convex with positive parameter `gamma_fconj`. Need to be sure that gamma_fconj = {} is the correct strongly convex constant".format(self.gamma_fconj))
         
-
         self.configured = True
         print("{} configured".format(self.__class__.__name__, ))
 
-    def update_previous_solution(self):
```

Comment:
This needs instigating. You commented out this step in the algorithm but it was there for a reason.
What made you believe this was unnecessary or wrong?